worldeditadditions/utils/parse/axes_parser.lua

```diff
@@ -201,4 +203,8 @@ function parse.keytable(tbl, facing, sum)
 	
 end
 
-return parse
+return {
+	keyword = parse.keyword,
+	keytable = parse.keytable,
+	number = parse.num,
```

Comment:
Much better, but if your `parse.num` function is being exported I believe given how generic it is it really should be in a separate file.

Reply:
Well it's required for the UAK parser to work, and it is a parser itself so.... I don't think so (I thought really hard about that when I made the function). Also I'd like to keep all of the UAK parser sub functions in the same place. Also also you can't have a keytable parser without a number parser.

Reply:
Right, got it. Let's keep this private then and not expose it, since it's an integral part of the new parser.

Reply:
Ok.